doc-assets/node/18005-search-videos/18005.html

```diff
@@ -348,52 +277,66 @@ <h2 id="Quoted_search_terms">Quoted search terms</h2>
     <p>Most browsers and other agents will treat literal quotation marks (<code>"..."</code>) correctly, but if you run into a case where quoted search terms do not appear to be returning correct results, try replacing the quotation marks with <code>%22</code>
       (<code>%22...%22</code>)</p>
 
-<pre class="line-numbers"><code class="language-http">q="foo" or q=%22foo%22
-q="foo+bar"or q=%22foo+bar%2</code>
-</pre>
+    <pre class="line-numbers">
+      <code class="language-http">
+        q="foo" or q=%22foo%22
+        q="foo%20bar" or q=%22foo%20bar%22
+      </code>
+    </pre>
 
     <aside class="bcls-aside bcls-aside--information">Note that the quotations marks require matching videos to have all the terms inside them, but not necessarily exact matches on those terms. For example, a search on <code>"home run"</code> might return a video with the words <code>homeless</code> and <code>running</code> in the title.</aside>
 
     <p>This also works when searching against a specific field:</p>
 
-<pre class="line-numbers"><code class="language-http">q=name:"home" q=name:"home+run" </code></pre>
+    <pre class="line-numbers">
+      <code class="language-http">
+        q=name:"home"
+        q=name:"home%20run"
+      </code>
+    </pre>
 
     <h3>Multiple words</h3>
 
     <p>Example: Notice that this request returns videos which have either a value of <code>sea</code> or <code>mammal</code> in the <code>tags</code> field.</p>
 
-<pre class="line-numbers"><code class="language-http">https://cms.api.brightcove.com/v1/accounts/921483702001/videos?q=tags:sea,mammal</code></pre>
+    <pre class="line-numbers"><code class="language-http">https://cms.api.brightcove.com/v1/accounts/921483702001/videos?q=tags:sea,mammal</code></pre>
 
     <p>But, the following request returns only those videos which have a tag <code>sea,mammal</code>.</p>
 
-<pre class="line-numbers"><code class="language-http">https://cms.api.brightcove.com/v1/accounts/921483702001/videos?q=tags:"sea,mammal"</code></pre>
+    <pre class="line-numbers"><code class="language-http">https://cms.api.brightcove.com/v1/accounts/921483702001/videos?q=tags:"sea,mammal"</code></pre>
 
-    <aside class="bcls-aside bcls-aside--warning">Tags are a special case where you can search for multiple tags as a comma-delimited list - that is not true for other fields like name, description, etc. For other kinds of multiple terms, combine terms with <code>+</code> sign, like this: <code>q=tags:"bird"+name:"owl"</code>.</aside>
+    <aside class="bcls-aside bcls-aside--warning">Tags are a special case where you can search for multiple tags as a comma-delimited list - that is not true for other fields like name, description, etc. For other kinds of multiple terms, combine terms with <code>+</code> sign, like this: <code>q=tags:"bird"%20name:"owl"</code>.</aside>
```

Comment:
I'd see if we can scratch any docs around comma separated tags so we can look at killing that feature.

Reply:
Actually we're stuck with this until after migration also since otherwise we can't support the "at least one must match" case